src/application-manager.js

```diff
@@ -225,34 +209,26 @@ export class ApplicationManager extends EventEmitter {
 		return this.emit('change', data);
 	}
 
-	init() {
-		return Images.initialized
-			.then(() => Images.cleanupDatabase())
-			.then(() => {
-				const cleanup = () => {
-					return docker.listContainers({ all: true }).then((containers) => {
-						return logger.clearOutOfDateDBLogs(_.map(containers, 'Id'));
-					});
-				};
-				// Rather than relying on removing out of date database entries when we're no
-				// longer using them, set a task that runs periodically to clear out the database
-				// This has the advantage that if for some reason a container is removed while the
-				// supervisor is down, we won't have zombie entries in the db
-
-				// Once a day
-				setInterval(cleanup, 1000 * 60 * 60 * 24);
-				// But also run it in on startup
-				return cleanup();
-			})
-			.then(() => {
-				return this.localModeManager.init();
-			})
-			.then(() => {
-				return this.services.attachToRunning();
-			})
-			.then(() => {
-				return this.services.listenToEvents();
+	async init() {
+		await Images.initialized;
+		await Images.cleanupDatabase();
+		const cleanup = () => {
+			return docker.listContainers({ all: true }).then((containers) => {
+				return logger.clearOutOfDateDBLogs(_.map(containers, 'Id'));
 			});
+		};
+		// Rather than relying on removing out of date database entries when we're no
+		// longer using them, set a task that runs periodically to clear out the database
+		// This has the advantage that if for some reason a container is removed while the
+		// supervisor is down, we won't have zombie entries in the db
+
+		// Once a day
+		setInterval(cleanup, 1000 * 60 * 60 * 24);
+		// But also run it in on startup
+		await cleanup();
+		await this.localModeManager.init();
+		await this.services.attachToRunning();
+		this.services.listenToEvents();
```

Comment:
should this not be a `return ...` ?

Reply:
Nah, it should be awaited though, strange, the editor said it didn't need it for whatever reason.